packages/edit-site/src/components/page-templates/dataviews-templates.js

```diff
@@ -59,6 +61,10 @@ const defaultConfigPerViewType = {
 		mediaField: 'preview',
 		primaryField: 'title',
 	},
+	[ LAYOUT_LIST ]: {
+		primaryField: 'title',
```

Comment:
Do you think the `primaryField` could be different between views? I can't think of a use case like that, so maybe we should add in the fields API(ex `isPrimary`)? 🤔

Reply:
maybe it can be modified, and in this case, it just becomes a top level config, like "sort" or "visibleFields" or something like that.

Reply:
> Do you think the primaryField could be different between views? I can't think of a use case like that, so maybe we should add in the fields API(ex isPrimary)? 🤔

I feel the same as you: it's quite improbable that it's different across views. However, I think the meaning of "primary" is something that belongs to the view API and not the field API.

> maybe it can be modified, and in this case, it just becomes a top level config, like "sort" or "visibleFields" or something like that.

By the user, you mean?

Reply:
I'd leave it as it is in this PR. If it requires further action, it can be done separately.